gluten-core/src/main/scala/io/glutenproject/execution/SortExecTransformer.scala

```diff
@@ -52,123 +51,36 @@ case class SortExecTransformer(
 
   override def output: Seq[Attribute] = child.output
 
-  override def outputOrdering: Seq[SortOrder] = sortOrder
+  override def outputOrdering: Seq[SortOrder] = child match {
```

Comment:
can we pass an `Option[AttributeMap[Alias]]` to SortExecTransformer if we have pulled out the pre-project ? if it is definded, we replace attribute with the alias, if not do nothing. So we do not need pre-project tag.